src/LitePsmJob.sol

```diff
@@ -0,0 +1,104 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Copyright (C) 2021 Dai Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity 0.8.13;
+
+import {IJob} from "./interfaces/IJob.sol";
+
+interface SequencerLike {
+    function isMaster(bytes32 network) external view returns (bool);
+}
+
+interface LitePsmLike {
+    function chug() external returns (uint256 wad);
+    function cut() external view returns (uint256 wad);
+    function fill() external returns (uint256 wad);
+    function gush() external view returns (uint256 wad);
+    function rush() external view returns (uint256 wad);
+    function trim() external returns (uint256 wad);
+    function gem() external returns (address);
+    function ilk() external returns (bytes32);
+    function vat() external returns (address);
+    function buf() external returns (uint256);
+    function buyGem(address usr, uint256 gemAmt) external returns (uint256 daiInWad);
+    function sellGem(address usr, uint256 gemAmt) external returns (uint256 daiOutWad);
+    function file(bytes32 what, uint256 data) external;
+}
```

Comment:
When using inline interfaces, usually we declare only the methods that are being used in the file.

Reply:
yeap, fixed in [092a8cd](https://github.com/makerdao/dss-cron/pull/25/commits/092a8cdcd7eb3470b95287314b863123169eb59e)